lib/src/base-styles/_nx-card.scss

```diff
@@ -14,91 +14,62 @@
   border-radius: $nx-border-radius;
   box-sizing: border-box;
   margin-bottom: $nx-spacing-md;
-  padding: $nx-spacing-l $nx-spacing-xxl;
-}
-
-.nx-card__call-out {
-  @include bold();
-
-  font-size: 32px;
-
-  .nx-icon--card-call-out {
-    font-size: 48px;
-  }
-
-  .nx-icon--card-call-out-xl {
-    font-size: 64px;
-  }
+  padding: $nx-spacing-l;
+  text-align: center;
 }
 
 .nx-card-container--row {
+  @include container-horizontal;
   display: flex;
-  -ms-flex-wrap: wrap;
   flex-wrap: wrap;
+  justify-content: space-between;
   margin-bottom: 0;
+}
 
-  + .nx-card-container--row {
-    margin-top: 0;
-  }
-
-  .nx-card {
-    @include container-vertical;
-    display: flex;
-    flex-direction: column;
-    -ms-flex-pack: justify;
-    justify-content: space-between;
-    margin-right: $nx-spacing-md;
-    text-align: center;
-    min-width: 230px;
-    max-width: 330px;
-  }
-
-  .nx-card__header {
-    @include container-vertical;
-    margin-bottom: $nx-spacing-md;
-  }
+.nx-card {
+  @include container-vertical;
+  display: grid;
+  grid-template-rows: auto 1fr auto;
+  margin-right: $nx-spacing-md;
+  min-width: 230px;
+  max-width: 330px;
+}
 
-  .nx-card__call-out {
-    + .nx-card__content {
-      margin-top: $nx-spacing-md;
-    }
-  }
+.nx-card__header {
+  @include container-vertical;
+  margin-bottom: $nx-spacing-md;
+}
 
-  .nx-card__call-out--text-only {
-    + .nx-card__content {
-      margin-top: $nx-spacing-xs;
-    }
-  }
+.nx-card__content {
+  align-content: start;
+  align-items: start;
```

Comment:
I'm fairly sure this doesn't do anything since the grid box heights are fit to the content anyway.

Reply:
> I'm fairly sure this doesn't do anything since the grid box heights are fit to the content anyway.

Yep: https://github.com/sonatype/sonatype-react-shared-components/pull/307/commits/c8367f79be644db6e71d67289f6e749e99d519b9